### ISSUE TITLE
Custom object detection

### ISSUE THREAD

**Issue by TanJohn1604** [dusty-nv/jetson-inference#1516]
Hi @dusty-nv 
I was following the link below. 
[https://github.com/dusty-nv/jetson-inference/blob/master/docs/pytorch-ssd.md](url)

at the Processing Images with TensorRT step i realised something

            IMAGES=<path-to-your-jetson-inference>/data/images   # substitute your jetson-inference path here
            
            detectnet --model=models/fruit/ssd-mobilenet.onnx --labels=models/fruit/labels.txt \
                      --input-blob=input_0 --output-cvg=scores --output-bbox=boxes \
                        "$IMAGES/fruit_*.jpg" $IMAGES/test/fruit_%i.jpg

1 ) i see  "detectnet --model=models/fruit/ssd-mobilenet.onnx .... " in above command. Can you explain what meaning of 'detecnet', becasue i cann't find detectnet.py in any folder.

2 ) In your pretrain file
[https://nvidia.box.com/shared/static/gq0zlf0g2r258g3ldabl9o7vch18cxmi.gz](url)

labels.txt
                      BACKGROUND
                      Apple
                      Banana
                      Grape
                      Orange
                      Pear
                      Pineapple
                      Strawberry
                      Watermelon

i saw your argument like this  ' --class-names "Apple,Orange,Banana,Strawberry,Grape,Pear,Pineapple,Watermelon"  '
it doesn't have 'BACKGROUND'
 So why it has 'BACKGROUND' in labels.txt

Hope your answers, thank you very much .

i am so sorry for my English is not good.

**Comment by dusty-nv** [comment]
> 1 ) i see "detectnet --model=models/fruit/ssd-mobilenet.onnx .... " in above command. Can you explain what meaning of 'detecnet', becasue i cann't find detectnet.py in any folder.

Hi @TanJohn1604, `detectnet` is the C++ version and `detectnet.py` is the Python version (these are [examples](https://github.com/dusty-nv/jetson-inference#code-examples) that come with the repo)

These get compiled and installed under `/usr/local/bin` when you run `make && sudo make install`, so you can run them from any directory.



> i saw your argument like this ' --class-names "Apple,Orange,Banana,Strawberry,Grape,Pear,Pineapple,Watermelon" '
> it doesn't have 'BACKGROUND'
> So why it has 'BACKGROUND' in labels.txt

The `BACKGROUND` class automatically gets added by train_ssd.py to reduce false positives
